core/templates/dev/head/base.html

```diff
@@ -118,7 +118,7 @@
                 <div class="navbar-container">
                   <div class="navbar-header protractor-test-navbar-header pull-left">
                     <a ng-if="isWindowNarrow" ng-click="openSidebar()" class="navbar-brand oppia-navbar-menu oppia-transition-200">
-                      <img src="/images/general/menu.png">
+                      <i class="material-icons menu">&#xE5D2;</i>
```

Comment:
Oh, I see. Nope -- please add a new class like along the lines of .oppia-navbar-menu-icon, and put that just below oppia-navbar-menu in the CSS file.